test.sh

```diff
@@ -63,7 +63,6 @@ else
   run go test ${dirlist}
 fi
 
-echo "Checking for unformatted files:"
```

Comment:
I stripped it because if there are no unformatted files, the test output ends with this statement, which is confusing. I'd be happy to add back in, so long as we end with an "ok" or something.